docs/workloads/azure/data/etl_pipelines/datastacks.md

```diff
@@ -0,0 +1,103 @@
+---
+id: datastacks
+title: Datastacks overview
+sidebar_label: Datastacks
+hide_title: false
+hide_table_of_contents: false
+description: Overview of the Datastacks utility
+keywords:
+  - data
+  - python
+  - etl
+  - cli
+  - azure
+  - template
+---
+
+Datastacks is a utility built to support various functions within the Ensono Stacks Data Platform. The library and its associated CLI is intended to assist developers working within a deployed Stacks Data Platform, supporting common tasks such as generating new data engineering workloads and running Spark jobs.
+
+## Using the Datastacks CLI
+
+* [Setup project environment](../getting_started/dev_quickstart_data_azure.md)
```

Comment:
you might want to test building it in parallel in Amido PR - I think the linting will fail without new line here